sycl/include/CL/sycl/detail/pi.h

```diff
@@ -1769,6 +1772,13 @@ __SYCL_EXPORT pi_result piextPluginGetOpaqueData(void *opaque_data_param,
 /// \param PluginParameter placeholder for future use, currenly not used.
 __SYCL_EXPORT pi_result piTearDown(void *PluginParameter);
 
+/// API to get Plugin specific warning and error messages.
+/// \param message is a returned address to the first element in the message.
```

Comment:
Please clarify in the API who owns the storage of the message and what it's lifetime is.

Reply:
I have clarified the storage and lifetime of the message string